umbraco/Umbraco-CMS
# V13: Change request column to NVarcharMax

This was reported by the DXP team.
They ran into issues using SqlServer, as when their content went over 255 characters, they would get errors when trying to send webhooks. This was because the RequestObject column was not set to Nvarcharmax

# How to test
- Create a document type with a richtext editor field, and allow as root set to true.
- Create a webhook with the URL set (i use https://webhook.site/ for testing) and Content is published event.
- Create some content with the document type in step 1, and fill up the rich text editor with more than 255 characters
- Save and publish
- Go to Settings -> Webhooks -> logs
- Click the one log you have (or the latest one) 
- It should now show a complete content in the `Request Body` 🙌
